server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -279,9 +267,8 @@ static boolean checkInit(VolumeManager fs, SiteConfiguration sconf, Configuratio
       return false;
     }
     if (sconf.get(Property.INSTANCE_SECRET).equals(Property.INSTANCE_SECRET.getDefaultValue())) {
-      LineReader c = getLineReader();
-      var w = c.getTerminal().writer();
-      c.getTerminal().puts(InfoCmp.Capability.bell);
+      Console c = System.console();
```

Comment:
I was more thinking of something like this: 

```java
      System.out.println();
      System.out.println();
      System.out.println("Warning!!! Your instance secret is still set to the default,"
          + " this is not secure. We highly recommend you change it.");
      System.out.println();
      System.out.println();
      System.out.println("You can change the instance secret in accumulo by using:");
      System.out.println("   bin/accumulo " + org.apache.accumulo.server.util.ChangeSecret.class.getName());
      System.out.println("You will also need to edit your secret in your configuration"
          + " file by adding the property instance.secret to your"
          + " accumulo.properties. Without this accumulo will not operate" + " correctly");


```

since all this is trying to do is print those messages to screen.

Reply:
Would we also be able to do `System.out.println("\n");` instead of two seprate println's? Or even appending two newlines (\n) to the preceding statements?

Reply:
Probably could do some cleanup here, yes.

Reply:
\n is unix if we do it the way  it is, its compatible with any other systems you run on. I suggest we live it like that